### DISCUSSION TITLE
Winlogbeat Cannot Edit winlogbeat.example.yml

### DISCUSSION THREAD

**Discussion by Ryan8bit** [Security-Onion-Solutions/securityonion#9558] [Unsupported Versions]
How do I edit winlogbeat.example.yml? I am trying to follow https://www.youtube.com/watch?v=Xz-7oDrZdQY
I am trying to run winlogbeat. How do I run winlogbeat?

**Comment by cm-ops** [comment]
Are you also following https://docs.securityonion.net/en/latest/beats.html#winlogbeat?

**Comment by InfosecGoon** [comment]
If you're having difficulty editing the winlogbeat.example.yml file to make winlogbeat.yml, the issue may be permissions-related; you need to be running the editor as an Administrator user to edit those files. Try running Wordpad as an Administrator, then open the files from inside the editor.

**Comment by Ryan8bit** [comment]
Where do I put the winlogbeat.yml file to actually use that file when I start the elastic winlogbeat oss service?

**Reply by Ryan8bit** [reply]
![WinlogbeatFailure](https://user-images.githubusercontent.com/106710009/212185410-142a7fec-3364-4288-abbd-7e20d649aca9.PNG)
I moved winlogbeat.yml to the desktop with the .msi file.
I had the elastic winlogbeat oss service service started. Then I stopped it. Then I started it again and it keeps failing. What do I do?

**Reply by InfosecGoon** [reply]
The winlogbeat.yml file needs to be in the same directory as the example (C:\programdata\Elastic\Beats\winlogbeat\winlogbeat.yml). 

Other things to check:

1. Make sure that you have used so-allow to open a firewall hole on your manager or standalone node on port 5044 so that the traffic from Winlogbeat will be accepted.

2. Make sure that your winlogbeat.yml has commented out the Elasticsearch output configuration that's in there by default, and uncommented the Logstash one, and put the proper IP address into it.

**Reply by Ryan8bit** [reply]
Which IP address is the proper one?

**Reply by Ryan8bit** [reply]
The VMnet8 IP address?

**Reply by InfosecGoon** [reply]
You're trying to send the Winlogbeat traffic to the Logstash container on your Security Onion server, so you would want to put the IP address of the Security Onion management interface into the Logstash output configuration in winlogbeat.yml.

**Reply by Ryan8bit** [reply]
How do I find out the IP address of the Security Onion management interface?

**Reply by Ryan8bit** [reply]
Is this the IP address using to log into the Security Onion Console?

**Reply by Ryan8bit** [reply]
Thanks both of you for your help!
The IP address used to log into the Security Onion Console is the correct ip address (the IP address of the Security Onion Management Node).
